pkg/diagnostics/log/log.go

```diff
@@ -0,0 +1,197 @@
+package log
```

Comment:
Put another way, this is a stateful output utility and stateful things should be instance wide, not execution environment wide.